Automattic/sensei
# Add Paid Course Upsell

Fixes #5129

### Changes proposed in this Pull Request

* Add the Paid Course upsell screen;
* Move the title from the steps in the Editor Wizard to the interior of their content;
* Add a way to add an image to the right side of each step, as per the designs;

### Testing instructions

1. Checkout this branch;
2. Guarantee that Sensei Pro isn't installed; (by the way, remember to backup your Sensei Pro directory if you're going to delete it through WordPress...hehe);
3. Create a new course;
4. Click "Next" on the modal that appears to see the Paid Course upsell screen;
5. Verify that the "Get Sensei Pro" button works and opens a new tab on the Sensei LMS page (with some UTM tracking codes in the query string, too). Also, verify that the editor wizard goes to the next step (the Course Pattern step);
6. Create another new course;
7. Click on the "Continue with Sensei Free" button and verify that it DOES NOT open a new tab, and just goes to the Course Pattern step;

### Screenshot of the current version

![Screenshot of the Paid Course Upsell screen](https://user-images.githubusercontent.com/529864/169920896-561ae2c1-8cb3-4c34-805b-d453b6ac78f3.png)

### To do

- [x] Fix price font size;
- [x] Adjust font style of the text "per year, 1 site" just below the price;
- [x] Add a background image covering both the content area of the modal + the header (I accept ideas and suggestions here! Still struggling a little bit with the CSS part..hehe) (**WIP**);
- [x] Add analytics parameters to the link opened when clicking on the "Get Sensei Pro" button?

### Questions

- I accept suggestions of tests to do; =)
- I accept any feedback about the quality of the code;
